src/renderer/root-frame.tsx

```diff
@@ -54,8 +54,17 @@ export class RootFrame extends React.Component {
     lensProtocolRouterRendererInjectable: LensProtocolRouterRenderer,
   ) {
     catalogEntityRegistry.init();
-    extensionLoader.loadOnClusterManagerRenderer();
+
+    try {
+      const loadingExtensions = extensionLoader.loadOnClusterManagerRenderer();
+      const loadingBundledExtensions = loadingExtensions.filter(e => e.isBundled).map(e => e.loaded);
+
+      await Promise.all(loadingBundledExtensions);
```

Comment:
Will we be stuck on the splash screen if a bundled extension gets stuck loading? A log message after some timeout might help with diagnosing the problem, if it happens to an end-user. Not sure if that is easy to do here, or somewhere else.

Reply:
Hmm, yeah that's a good point. It shouldn't block if extension fails but if it's stuck loading will do that. I think a timeout should be doable. It's not end of the world even if we continue and the extension keeps loading. I wonder what's a good time out, 10 seconds?

Reply:
Yes, having a slightly worse UX (what we have now) when an extension takes too long (which really shouldn't happen for bundled ones) is better than being stuck on the splash screen. I would think 10 seconds from the time the extensions start loading is adequate.

Reply:
I added a 10 second timeout now. I tested it with copying one of the extensions from lens-ide repo to the `lens/extensions`, and added 20 second delay on the `onActivate`-method on `renderer.tsx`. `make build` and open the app.